lib/index.js

```diff
@@ -74,6 +74,14 @@ HubSpot.prototype.identify = function(identify) {
 
 HubSpot.prototype.track = function(track) {
   var props = track.properties();
+
+  // Hubspot expects properties.id to be the name of the .track() event
+  // Ref: http://developers.hubspot.com/docs/methods/enterprise_events/javascript_api
+  if (props.id) {
+    props._id = props.id;
+    props.id = track.event();
```

Comment:
Doesn't this mean that if the user doesn't specify a `properties.id`, then the event name will never be populated at `id`?